README.md

```diff
@@ -1,70 +1,90 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Capstone React Project 
 
-## Available Scripts
+> This is the final Capstone React Project using Nelson Sakwa on Behance design to complete 2 home and details module and display the metrics number in the project. Using the API, the project displays the US population total number and categorized by states which are further categorized by county. 
 
-In the project directory, you can run:
 
-### `npm start`
+## Built With
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- HTML, CSS, Web Pack, Javascript
+- Visual Studio Code and GitHub
+- Git Flow, Webpack, Jest
+- React JS
+- React Redux 
+- React Logger
+- React Router
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Live Demo (if available)
+[Live Preview](https://6311cafbdda003000ae2de4b--glistening-biscotti-7c4251.netlify.app/)
 
```

Comment:
- [ ] Kindly add your presentation link just like you did for the live link, and in the video please mention the part in the project that you are most proud of.
![Screen Shot 2022-09-02 at 1 02 37 PM](https://user-images.githubusercontent.com/59416136/188115888-0fa747a5-35a6-40e3-af91-858cac718890.png)

Reply:
I will make another video to display the implementation on netlify.

Reply:
https://www.loom.com/share/5c1c68f39bf145d2b4407885603fbf2e

Reply:
@Wes-Isaac I made another loom video